f5router/f5router.go

```diff
@@ -438,6 +483,29 @@ func (r *F5Router) createiRules(pm bigipResources.PartitionMap, partition string
 	}
 }
 
+func (r *F5Router) createMonitors(pm bigipResources.PartitionMap, partition string, wg *sync.WaitGroup) {
+	defer wg.Done()
+
+	for _, monitors := range r.monitorResources {
```

Comment:
The bind and unbind in combination with route add and remove should be keeping the r.monitoResources up to date, this should be similar to the other create* functions and just transfer monitors into the pm